core/dispatch_json.js

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+((window) => {
```

Comment:
I would prefer if we could support ESM usage, but if we have to chose one it clearly makes more sense to favor simpler use cases.